validation/validation.go

```diff
@@ -61,6 +61,19 @@ func check(value any, p tree.Path) error {
 	return nil
 }
 
+func checkSecrets(value any, p tree.Path) error {
+	if err := checkFileObject("file", "environment")(value, p); err != nil {
+		return err
+	}
+
+	v := value.(map[string]any)
+	if _, ok := v["content"]; ok {
+		return fmt.Errorf("%s: content is not allowed for secrets", p)
```

Comment:
It doesn't make sense to have this made valid by schema but rejected by parser.

Reply:
so I see only one way - to revert your changes where you added "content" to the "secrets" property

Reply:
@ndeloof if you have any other ideas, please share